docs/network/fees.md

```diff
@@ -27,23 +27,23 @@ If the transaction's base fee is not enough to cover Etherlink's fees, the trans
 
 The execution fee changes based on the transaction throughput over time.
 
-Etherlink measures throughput in terms of _ticks_, which are execution steps.
-As demand increases, Etherlink runs more ticks per second to process more transactions in the same amount of time.
+Etherlink measures throughput in terms of the amount of execution gas (that is, the units of computation used to execute transactions) used over time.
+As demand increases and more execution gas is used in a shorter time, Etherlink increases the gas price to increase the execution fee.
 
-Etherlink has a target number of ticks to run per second, referred to as the _speed limit_.
-To calculate the throughput, Etherlink stores the number of ticks used by transactions and subtracts the speed limit every second.
-The remaining number of ticks is known as the _backlog_.
+Etherlink has a target amount of execution gas to use per second, referred to as the _speed limit_.
+To calculate the throughput, Etherlink records the total execution gas used and subtracts the speed limit every second.
+The remaining execution gas amount is known as the _backlog_.
 
-In this way, if the number of ticks that transactions use per second exceeds the speed limit, the backlog increases and the execution fee goes up.
-If the number of ticks they use per second is below the speed limit, the backlog decreases or reaches zero and the execution fee decreases or reaches a base amount.
+In this way, if the execution gas used per second exceeds the speed limit, the backlog increases and the execution fee can go up.
```

Comment:
The gas will increase if the gas consumption exceeds the limit for a certain amount of time, I'm not sure if we can add this information or how we can formulate it.